Dockerfile

```diff
@@ -1,17 +1,18 @@
-FROM node:16
+FROM node:19
 
+ENV TERM=xterm
+RUN mkdir /usr/src/app
 WORKDIR /usr/src/app
 
 COPY package*.json ./
-
 RUN npm install
 
-COPY . .
-
+COPY .env ./
+COPY tsconfig.json ./
+COPY src ./src
 RUN npm run build
-RUN mkdir /logs
-
-VOLUME ["/logs:/logs"]
-CMD ["node", "out/index.js", " > ", "logs/log.txt"]
 
+RUN mkdir /logs
+VOLUME ["/logs"]
```

Comment:
would this mean we are able to check console logs outside of the container? that is something that i want to add for a while

Reply:
It is the start, still need to do some mounting and such to actually get it to work.  I was interested in that as well and was going to get that work.  This PR is just the basics.

Reply:
Seems like it is just easier to just dump into a file with the docker logs command. Mounting is just too finicky IMHO.  Example: `docker logs white-whale-bot > log.txt`